contrib/pkg/createcluster/create.go

```diff
@@ -392,6 +402,42 @@ func (o *Options) Validate(cmd *cobra.Command) error {
 			return fmt.Errorf("unable to parse key=value label: %s", ls)
 		}
 	}
+
+	if len(o.HttpProxy) > 0 || len(o.HttpsProxy) > 0 {
+		if o.HttpProxy == "" && o.HttpsProxy == "" {
+			cmd.Usage()
+			log.Info("must set --http-proxy or --https-proxy")
+			return fmt.Errorf("must set --http-proxy or --https-proxy")			
+		}
+	}
+		
+	if len(o.HttpProxy) > 0 && o.HttpProxy != "" {
```

Comment:
Check for empty string seems to duplicate the length check, I think just len() > 0 is sufficient and correct.